package.json

```diff
@@ -41,12 +41,12 @@
     "express": "^4.14.0",
     "got": "^11.6.2",
     "prettier": "2.1.1",
-    "prom-client": "^12.0.0",
+    "prom-client": "^14.1.1",
     "ts-node": "^10.9.1",
     "typescript": "^4.8.2"
   },
   "peerDependencies": {
-    "prom-client": "12.x"
+    "prom-client": "12.x | 13.x | 14.x"
```

Comment:
We can do a `>/<` type thing.

This felt easier to understand if we want to support all of v14

Reply:
OK seeing different syntax on https://semver.npmjs.com/

Maybe I'll make it `12 - 14`